src/panels.py

```diff
@@ -180,6 +180,10 @@ class SequenceImportSettings(bpy.types.PropertyGroup):
         name="Relative Paths",
         description="Store relative paths for Streaming sequences and for reloading Cached sequences",
         default=True)
+    showAsSingleMesh: bpy.props.BoolProperty(
+        name='Show as Single Mesh',
```

Comment:
Could you change this from `Show as Single Mesh` to `Enable Alembic Export`?

I'm fine with leaving the property as `showAsSingleMesh` since it tells me something about the implementation, but I don't think the single-mesh thing will mean anything to users.

Reply:
Yes this is a good idea!